go.sum

```diff
@@ -0,0 +1,63 @@
+github.com/Azure/go-ansiterm v0.0.0-20170929234023-d6e3b3328b78 h1:w+iIsaOQNcT7OZ575w+acHgRric5iCyQh+xv+KJ4HB8=
+github.com/Azure/go-ansiterm v0.0.0-20170929234023-d6e3b3328b78/go.mod h1:LmzpDX56iTiv29bbRTIsUNlaFfuhWRQBWjQdVyAevI8=
```

Comment:
Why Azure?

Reply:
go.sum is generated automatically.
result of [go mod graph](https://golang.org/cmd/go/#hdr-Print_module_requirement_graph) : 

```
$ GO111MODULE=on go mod graph
github.com/skanehira/docui github.com/fsouza/go-dockerclient@v1.3.3
github.com/skanehira/docui github.com/jroimartin/gocui@v0.4.0
github.com/skanehira/docui github.com/mattn/go-runewidth@v0.0.4
github.com/skanehira/docui github.com/nsf/termbox-go@v0.0.0-20190104133558-0938b5187e61
github.com/skanehira/docui github.com/skanehira/gocui-component@v0.0.0-20190103120946-5f5ba7f77c3b
github.com/skanehira/docui github.com/stretchr/testify@v1.3.0
github.com/skanehira/docui golang.org/x/sys@v0.0.0-20190115152922-a457fd036447
github.com/stretchr/testify@v1.3.0 github.com/davecgh/go-spew@v1.1.0
github.com/stretchr/testify@v1.3.0 github.com/pmezard/go-difflib@v1.0.0
github.com/stretchr/testify@v1.3.0 github.com/stretchr/objx@v0.1.0
github.com/fsouza/go-dockerclient@v1.3.3 github.com/Azure/go-ansiterm@v0.0.0-20170929234023-d6e3b3328b78
github.com/fsouza/go-dockerclient@v1.3.3 github.com/Microsoft/go-winio@v0.4.11
github.com/fsouza/go-dockerclient@v1.3.3 github.com/Nvveen/Gotty@v0.0.0-20120604004816-cd527374f1e5
github.com/fsouza/go-dockerclient@v1.3.3 github.com/containerd/continuity@v0.0.0-20181203112020-004b46473808
github.com/fsouza/go-dockerclient@v1.3.3 github.com/docker/docker@v0.7.3-0.20180827131323-0c5f8d2b9b23
github.com/fsouza/go-dockerclient@v1.3.3 github.com/docker/go-connections@v0.4.0
github.com/fsouza/go-dockerclient@v1.3.3 github.com/docker/go-units@v0.3.3
github.com/fsouza/go-dockerclient@v1.3.3 github.com/docker/libnetwork@v0.8.0-dev.2.0.20180608203834-19279f049241
github.com/fsouza/go-dockerclient@v1.3.3 github.com/gogo/protobuf@v1.2.0
github.com/fsouza/go-dockerclient@v1.3.3 github.com/google/go-cmp@v0.2.0
github.com/fsouza/go-dockerclient@v1.3.3 github.com/gorilla/context@v1.1.1
github.com/fsouza/go-dockerclient@v1.3.3 github.com/gorilla/mux@v1.6.2
github.com/fsouza/go-dockerclient@v1.3.3 github.com/opencontainers/go-digest@v1.0.0-rc1
github.com/fsouza/go-dockerclient@v1.3.3 github.com/opencontainers/image-spec@v1.0.1
github.com/fsouza/go-dockerclient@v1.3.3 github.com/opencontainers/runc@v0.1.1
github.com/fsouza/go-dockerclient@v1.3.3 github.com/pkg/errors@v0.8.1
github.com/fsouza/go-dockerclient@v1.3.3 github.com/sirupsen/logrus@v1.3.0
github.com/fsouza/go-dockerclient@v1.3.3 github.com/vishvananda/netlink@v1.0.0
github.com/fsouza/go-dockerclient@v1.3.3 github.com/vishvananda/netns@v0.0.0-20180720170159-13995c7128cc
github.com/fsouza/go-dockerclient@v1.3.3 golang.org/x/crypto@v0.0.0-20190103213133-ff983b9c42bc
github.com/fsouza/go-dockerclient@v1.3.3 golang.org/x/sys@v0.0.0-20190102155601-82a175fd1598
github.com/fsouza/go-dockerclient@v1.3.3 gotest.tools@v2.2.0+incompatible
github.com/sirupsen/logrus@v1.3.0 github.com/davecgh/go-spew@v1.1.1
github.com/sirupsen/logrus@v1.3.0 github.com/konsorten/go-windows-terminal-sequences@v1.0.1
github.com/sirupsen/logrus@v1.3.0 github.com/pmezard/go-difflib@v1.0.0
github.com/sirupsen/logrus@v1.3.0 github.com/stretchr/objx@v0.1.1
github.com/sirupsen/logrus@v1.3.0 github.com/stretchr/testify@v1.2.2
github.com/sirupsen/logrus@v1.3.0 golang.org/x/crypto@v0.0.0-20180904163835-0709b304e793
github.com/sirupsen/logrus@v1.3.0 golang.org/x/sys@v0.0.0-20180905080454-ebe1bf3edb33
```

Reply:
@d-kuro 
Nice. it's best practice
thank's